### ISSUE TITLE
User story - Project tech association

### ISSUE THREAD

**Issue by stephendawsondev** [stephendawsondev/StackPortfolio#22]
### If applicable, add a related Epic.

Epic: #9 

### User Story

As a **tech user**, I can **add technologies used on my projects** so that I can **have them linked to my profile and show that I have evidence for those skills**.

### Acceptance Criteria

- When a user adds a new tech, it should be pending for admin approval (to avoid duplicates)
- New tech should appear as greyed out with an explanation as to why
- When tech is added to the project, it is added to the user's profile if not already there.
- When a project is removed and that tech is not present on other projects, it will remove it from the user's profile.

### Please ensure you've completed these tasks before marking the issue as done.

- [ ] Reviewed the user story for clarity and completeness.
- [ ] Identified and linked related issues or epics.
- [ ] Created or updated the necessary documentation.
- [ ] Written unit and/or integration tests for functionality.
- [ ] All code changes have been manually tested to ensure they meet the acceptance criteria.
- [ ] Updated the issue with any relevant notes or feedback (to date).
- [ ] Merged into the main branch and deployed to production (if applicable).

### Please add any further tasks that need to be completed before the issue can be marked as done.

- [ ] Create a table for the tech so that tech can be stored and unique
- [ ] Ensure intermediary tables are set up to work with Users, Projects and Job Posts.
- [ ] Load initial tech in

**Comment by stephendawsondev** [comment]
Closing as duplicate of #10
